test/integration-legacy/selenium-helpers.js

```diff
@@ -5,7 +5,7 @@ require('chromedriver');
 const headless = process.env.SMOKE_HEADLESS || false;
 const remote = process.env.SMOKE_REMOTE || false;
 const ci = process.env.CI || false;
-const buildID = process.env.TRAVIS_BUILD_NUMBER;
+const buildID = process.env.CIRCLE_WORKFLOW_ID;
```

Comment:
Maybe this is a good time to use something platform-agnostic like `BUILD_ID` for this. Then do the assignment a level up. Also, maybe it should have a recognizable default (like `0000` or something)?